debug_toolbar/toolbar.py

```diff
@@ -72,6 +72,14 @@ def csp_nonce(self):
         """
         return getattr(self.request, "csp_nonce", None)
 
+    @property
+    def health_level(self):
+        """
+        Return the maximum health level across all panels.
+        This is used to color the toolbar hidden button.
+        """
+        return max(panel.health_level for panel in self.panels)
```

Comment:
What happens when `self.panels` is empty? Is it possible? I think you might want to provide a default value as a fallback (`max((...), NONE)`)